ci/jenkins/test-vm.sh

```diff
@@ -303,12 +307,8 @@ function deliver_antrea_vm {
     export GOROOT=/usr/local/go
     export GOCACHE=${WORKSPACE}/../gocache
     export PATH=${GOROOT}/bin:$PATH
-    TEMP_ANTREA_TAR="antrea-image.tar"
 
     make docker-bin
```

Comment:
Do you know why we run `make docker-bin` after running `./hack/build-antrea-linux-all.sh` in apply_antrea ?